drivers/input/touchscreen/edt-ft5x06.c

```diff
@@ -1384,11 +1409,12 @@ static int edt_ft5x06_ts_probe(struct i2c_client *client,
 			return error;
 		}
 	} else {
+		tsdata->init_td_status = UINT_MAX; /* filter bogus initial data */
 		INIT_WORK(&tsdata->work_i2c_poll,
 			  edt_ft5x06_ts_work_i2c_poll);
 		timer_setup(&tsdata->timer, edt_ft5x06_ts_irq_poll_timer, 0);
 		tsdata->timer.expires = jiffies +
-					msecs_to_jiffies(POLL_INTERVAL_MS);
+			msecs_to_jiffies(FIRST_POLL_DELAY_MS);
```

Comment:
The previous indentation looks better.

Reply:
That was emacs' default for linux style - but I've just noticed that checkpatch allows both (surprisingly). I can increase it again.

Reply:
Would a signed integer and -1 be a nicer way to represent "uninitialized"?  Actual values are u8, and u8->int casting is presumably uncontroversial.

To anticipate another potential question: we deliberately do include the "reserved" upper nybble in the test, because it does help to discriminate between bogus and real touch data.

Reply:
Since `init_td_status` is a signed `int`, and the `rdbuf` values are `u8`s, making the magic values negative is clearer and guaranteed to work.